### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by RymMichaut**
Updating titles, giving descriptions and keywords to each technical content

### CODE REVIEW

File: docs/tutorials/index.md [hunk 1]
```diff
@@ -5,33 +5,33 @@ Learn how to use Taipy and improve your skills with the following tutorials.
...
+| 8   | Organize your applications with Partials, Dialogs, Panes |  Explore the power of Partials in Taipy front-end with this video tutorial. Learn how to streamline development, enhance clarity, and create efficient GUI pages and components. Discover the utility of Dialogs and Panes for an improved user interface. |  Organizing Taipy applications, Partials, Dialogs, Panes  | [organizing an advanced application](https://www.taipy.io/project/organizing-taipy-application-partials-dialogs/) |
+| 9   |  GS with Taipy Dataflow Orchestration (to merge with #2?)|  Learn the essentials of building multi-scenario pipeline-driven applications with Taipy back-end entities in this informative video tutorial. Explore data nodes, tasks, pipelines, and scenarios as they collaborate in managing complex data-driven applications. Dive into a hands-on demo using a diamond dataset, where machine learning models predict prices based on feature subsets. Create and manage multiple scenarios effortlessly with Taipy!   |  Get Started with Dataflow Orchestration  |  [GS with scenario mgt](https://www.taipy.io/project/getting-started-with-taipy-core/) |
```
Comment (FlorianJacta) [inline docs/tutorials/index.md]: I won't use the word Getting Started (GS)

File: docs/tutorials/index.md [hunk 2]
```diff
@@ -5,33 +5,33 @@ Learn how to use Taipy and improve your skills with the following tutorials.
...
+| 17  | Multi-page application |  Say goodbye to cluttered dashboards! Discover the power of multipage applications with Taipy. Organize your data and visualizations effortlessly, creating user-friendly and intuitive dashboards for enhanced insights. Simplify your data visualization journey today!   |  Multi-page application  |     [Multi-page application](https://www.taipy.io/tips/multipage-application/) |
+| 18  | Learn about Using tables |  Discover the power of Taipy front-end tables – your go-to for data presentation and control. Learn key settings and features in this article, tailored for effortless table creation within data applications. Simplify your data management with Taipy today!   |  Tables  |     [Learn about Using tables](https://www.taipy.io/tips/using-tables/) |
```
Comment (FlorianJacta) [inline docs/tutorials/index.md]: No capital for 'U' in 'Learn about Using tables'

File: docs/tutorials/index.md [hunk 3]
```diff
@@ -5,33 +5,33 @@ Learn how to use Taipy and improve your skills with the following tutorials.
...
 |:----|:----------------------------:|:---------------------------------------------------------------------------------:|:----------:|---------------------------------------------------------------------------------:|
-| 1   |      Understanding GUI       | Understand basic knowledge of Taipy GUI creating of a multi-page NLP application. |  **TODO**  |                     [gui tuto](complete_applications/understanding_gui/index.md) |
-| 2   | Scenario management Overview |             Learn the main features of the Taipy scenario management              |  **TODO**  | [scenario mgt tuto](complete_applications/scenario_management_overview/index.md) |
-| 3   | First Realistic application  |              Build a complete application from back-end to front-end              |  **TODO**  | [Complete application tuto](complete_applications/complete_application/index.md) |
+| 1   |      Understanding GUI       | Gain a fundamental understanding of creating a multi-page NLP application using Taipy front-end. |  Application front-end building  |                     [gui tuto](complete_applications/understanding_gui/index.md) |
```
Comment (FlorianJacta) [inline docs/tutorials/index.md]: I am not sure about this title. Maybe "Gain a fundamental understanding of Taipy front-end by creating a multi-page NLP application"?

**Review by jrobinAV — APPROVED** [review]
Looks good to me.

**Pull request merged**